client/task-list/tasks/payments/PaymentGatewaySuggestions/components/Setup/Setup.js

```diff
@@ -122,22 +126,25 @@ export const Setup = ( {
 			: null;
 	}, [ needsPluginInstall ] );
 
-	const connectStep = {
-		key: 'connect',
-		label: sprintf(
-			__( 'Connect your %(title)s account', 'woocommerce-admin' ),
-			{
-				title,
-			}
-		),
-		content: paymentGateway ? (
-			<Connect
-				markConfigured={ markConfigured }
-				paymentGateway={ paymentGateway }
-				recordConnectStartEvent={ recordConnectStartEvent }
-			/>
-		) : null,
-	};
+	const connectStep = useMemo(
+		() => ( {
+			key: 'connect',
+			label: sprintf(
+				__( 'Connect your %(title)s account', 'woocommerce-admin' ),
+				{
+					title,
+				}
+			),
+			content: gatewayInstalled ? (
+				<Connect
+					markConfigured={ markConfigured }
+					suggestedGateway={ suggestion }
```

Comment:
```suggestion
					gatewaySuggestion={ suggestion }
```

Or just `suggestion` here to keep things semantically similar.